app/renderer/settings.js

```diff
@@ -93,6 +93,7 @@ const defaultSettings = {
     "favicons": "session",
     "favoritepages": "",
     "firefoxmode": "never",
+    "followelement": "url,onclick,inputs-insert,inputs-click,other",
```

Comment:
This line is all good, but as this is a list of values that has specific values being valid for it, could you check for validity for values entered with `:set` by doing something similar as is done for modifiers on line 416 for example?